README.md

```diff
@@ -240,3 +240,41 @@ There are many differences between SQL syntax and functions supported by differe
 - PostgreSQL: `Database.Esqueleto.PostgreSQL`
 
 In order to use these functions, you need to explicitly import their corresponding modules.
+
+
+### Tests and Postgres
+
+To ```stack test``` with Postgresql you'll need to set the Postgres flag to true in esqueleto.cabal 
+
+```
+Flag postgresql
+  Description: test postgresql. default is to test sqlite.
+  Default: False
+```
+
+If you don't have Postgres install it. Using apt-get it's just:
```

Comment:
Add instructions for other users maybe? I'm thinking `homebrew` or else link to postgres site itself.